### PR TITLE
papers: 46.2 -> 47.0

### PR DESCRIPTION
**Pull request by honnip**
Website: https://gitlab.gnome.org/GNOME/Incubator/papers

Changelog:
- https://gitlab.gnome.org/GNOME/Incubator/papers/-/tags/47.0
- https://gitlab.gnome.org/GNOME/Incubator/papers/-/tags/47.rc
- https://gitlab.gnome.org/GNOME/Incubator/papers/-/tags/47.beta
- https://gitlab.gnome.org/GNOME/Incubator/papers/-/tags/47.alpha

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by honnip** [comment]
@ofborg build papers

**Review by FliegendeWurst — APPROVED** [review]

File: pkgs/by-name/pa/papers/package.nix [hunk 1]
```diff
@@ -46,19 +48,24 @@ stdenv.mkDerivation (finalAttrs: {
...
+  cargoRoot = "shell-rs";
 
-    outputHashes = {
-      "cairo-rs-0.20.0" = "sha256-us8Q1cqHbs0wSfMcRgZx7wTqSagYkLv/aNI8Fle2nNk=";
-      "gdk4-0.9.0" = "sha256-a+fkiCilKbq7sBHZ9Uvq9a/qqbsVomxG6K07B5f4eYM=";
-      "libadwaita-0.7.0" = "sha256-gfkaj/BIqvWj1UNVAGNNXww4aoJPlqvBwIRGmDiv48E=";
-    };
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) src pname version;
+    sourceRoot = "${finalAttrs.src.name}/${finalAttrs.cargoRoot}";
```
Comment (honnip) [inline pkgs/by-name/pa/papers/package.nix:67]: https://github.com/NixOS/nixpkgs/pull/350541

Comment (Aleksanaa) [reply pkgs/by-name/pa/papers/package.nix:67]: Please update

Comment (honnip) [reply pkgs/by-name/pa/papers/package.nix]: thank you

**Review by Aleksanaa — APPROVED** [review]

**Comment by Aleksanaa** [comment]
@ofborg build papers

**Comment by FliegendeWurst** [comment]
@ofborg build papers

(503 response)

**Comment by Aleksanaa** [comment]
Still

**Comment by jtojnar** [comment]
Might be better to switch to tarballs from the `gnome://` mirror. Though that will not help with the patch.

**Comment by honnip** [comment]
@ofborg build papers

**Pull request merged**